packages/data-service/README.md

```diff
@@ -0,0 +1,79 @@
+This package is to be extracted to dcos-ui/data-service.
```

Comment:
There is a PR for discussions on this file:
https://github.com/dcos/dcos-ui/pull/2841

Reply:
But why is it also in this PR 🤔

Reply:
I rebased from the same place to get all the web pack and other niceties. I was betting that by the time this one goes in, the other one would also have been in

Reply:
I put this on hold merge until the other one is merged.

Reply:
Actually, I will also rebase to not have that commit.